Open Application draft (#56)
DaanVanVugt/ignitioncomputing: _posts/careers/2024-09-03-Open-Application.md

---
title: Research Software Engineer
layout: featured
author: Yonis le Grand  
image: /assets/img/news/research_software_engineer.jpg
categories: careers
description: Open Application
---

### Open Application

> We're always excited to connect with talented physics software developers, computer scientists, simulation engineers, and those with expertise in nuclear fusion, engineering mathematics, high-performance computing or physics. If you're passionate about shaping the future of science through innovative software, we'd love to hear from you.

Author: ```suggestion
We're always happy to connect with talented physics software developers, computer scientists, simulation engineers, and those with expertise in nuclear fusion, engineering mathematics, high-performance computing, or physics. If you're excited about shaping the future of science through innovative software, we'd love to hear from you.
```

> We're always excited to connect with talented physics software developers, computer scientists, simulation engineers, and those with expertise in nuclear fusion, engineering mathematics, high-performance computing or physics. If you're passionate about shaping the future of science through innovative software, we'd love to hear from you.

Author: ```suggestion
Are you passionate about complex problem-solving,  high-tech systems, and software engineering? We're always happy to connect with talented physics software developers, computer scientists, simulation engineers, and those with expertise in nuclear fusion, engineering mathematics, high-performance computing, or physics. If you're excited about shaping the future of science through innovative software, we'd love to hear from you.
```

> We are continuosly looking for physics software developers, simulation software engineers, experts in high-performance and/or parallel computing, people with a background in nuclear fusion, engineering mathematics, computer science and physics. Do you fit one of these descriptions? Do you want to help build software for the future of science? Whether you are looking for a full-time role, internship or traineeship, feel free to

Reviewer: typo, continuously. Also, 'we are continuously looking' feels a bit desperate? I would try to rephrase things from the applicant's perspective, to indicate that we have a lot of interesting work, so it might be worthwhile to make an open application

> We are continuosly looking for physics software developers, simulation software engineers, experts in high-performance and/or parallel computing, people with a background in nuclear fusion, engineering mathematics, computer science and physicists. Do you fit one of these descriptions? Do you want to help build software for the future of science? Whether you are looking for a full-time role, internship or traineeship, feel free to

Reviewer 2: ```suggestion
We are continuosly looking for physics software developers, simulation software engineers, experts in high-performance and/or parallel computing, people with a background in nuclear fusion, engineering mathematics, computer science and physics. Do you fit one of these descriptions? Do you want to help build software for the future of science? Whether you are looking for a full-time role, internship or traineeship, feel free to
```

send us a message <careers@ignitioncomputing.com> with your ideas, CV and motivation letter.

> ### About US

Reviewer 2: ```suggestion
### About Us
```

Ignition Computing is a startup in Eindhoven based on the Keizersgracht. We specialize in high-performance computing and research software, with a focus on fusion energy. As a developer at Ignition Computing you will provide programming and consulting services for commercial and research support purposes. We mainly code in Python, C++, and Fortran. If you want to grow your skills as a developer, you're the kind of person we're after.

> Our culture is focussed on cooperation, growth and a healthy work-life balance. We are open to flexible work-arrangements such as (partial) work from home and Payed Time Off (PTO). At Ignition Computing we believe in an informal environment where everyone is equal. We expect our employees to contribute with new ideas and work in close cooperation with our team and our customers. We share our office, situated in the lively city centre of Eindhoven, with Afterservice. We have lunch and Friday drinks together.

Reviewer 2: _We're open to PTO_ sounds a bit odd 🤔

#### Some of the current running projects we have:

- Preconnet, a tool to benchmark and accelerate physics simulations through optimizing solver configurations, initial guesses and preconditioning.

> - [ITER](https://iter.org), the world's biggest nuclear fusion project where we have multiple running projects such as working on the underlying data structures for experimental data ([IMASPy](https://pypi.org/project/imaspy/)) and developing simulation codes.

Reviewer: the imaspy popen-source project is years behind the 'real' one (and we should probably make a notice on the projectp age)

> - Consulting for various Dutch tech companies such as [Celsian](https://www.celsian.nl/) and [Plasma Matters](https://plasimo.phys.tue.nl/index.php).

Reviewer: CelSian

> - Consulting for various Dutch tech companies such as [Celsian](https://www.celsian.nl/) and [Plasma Matters](https://plasimo.phys.tue.nl/index.php).

Author: ```suggestion
- Consulting for various Dutch tech companies such as [CelSian](https://www.celsian.nl/) and [Plasma Matters](https://plasimo.phys.tue.nl/index.php).
```

> - Consulting for an American fusion start-up (currently in stealth).

Reviewer: stealth mode

> To apply, send your job application, which includes CV and cover letter, to <careers@ignitioncomputing.com>

Reviewer: I think we can avoid the cover letter, I'd be more interested in a work sample

> To apply, send your job application, which includes CV and cover letter, to <careers@ignitioncomputing.com>

Author: ```suggestion
To apply, send your job application, which includes CV and a salient coding sample (e.g., GitHub repository, code snippet, or project portfolio) , to <careers@ignitioncomputing.com>
```
